packages/adapter-drizzle/src/drivers/sqlite.ts

```diff
@@ -80,7 +81,10 @@ export class DrizzleSQLiteAdapter implements Adapter {
 			.values({
 				id: session.id,
 				userId: session.userId,
-				expiresAt: Math.floor(session.expiresAt.getTime() / 1000),
+				// If the drizzle schema is set to `mode: "timestamp"` drizzle will already expect a date from us so we shouldn't convert
+				expiresAt: is(this.sessionTable.expiresAt, SQLiteTimestamp)
```

Comment:
Do you know how reliable this check is? Wonder if it's an `instanceof` check or comparing its properties

Reply:
Looks like it's either a check for a property with a symbol key of uses the name (they recommend over instanceof guessing because of the issues that brings when making multiple bundles):

https://github.com/drizzle-team/drizzle-orm/blob/15ff6b49bff580b8f07077a044be4cb52ec17183/drizzle-orm/src/entity.ts#L12
https://orm.drizzle.team/docs/goodies#compare-objects-types-instanceof-alternative

Reply:
Will the `Symbol` comparison work across different versions of Drizzle?

Reply:
They use [`Symbol.for`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/for) so assuming they don't change the key name then yes it will

Reply:
Missed that part, thanks!